crates/outbound-http/src/lib.rs

```diff
@@ -1,179 +1,12 @@
 pub mod allowed_http_hosts;
+#[cfg(feature = "runtime")]
 mod host_component;
+#[cfg(feature = "runtime")]
+mod outbound_http;
```

Comment:
The module name is a bit awkward since it's also the name of the crate. Perhaps `host_impl`?

Reply:
Your nit is my command - done!